### PR TITLE
paint.cc: Simplify drawing line

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
This touches the same function as #1653. If they're both relevant, can you merge them?

**Comment by deanlee** [comment]
merged to https://github.com/commaai/openpilot/pull/1653

**Comment by deanlee** [comment]
#1653 is closed.so I reopened this PR

File: selfdrive/ui/paint.cc [hunk 1]
```diff
@@ -213,20 +205,12 @@ static void update_all_track_data(UIState *s) {
...
 
 static void ui_draw_track(UIState *s, bool is_mpc, track_vertices_data *pvd) {
+ if (pvd->cnt == 0) return;
+
   nvgBeginPath(s->vg);
-  bool started = false;
-  for(int i = 0;i < pvd->cnt;i++) {
-    float x = pvd->v[i].x;
-    float y = pvd->v[i].y;
-    if (x < 0 || y < 0) {
```
Comment (deanlee) [inline selfdrive/ui/paint.cc:221]: x,y always >=0

Comment (adeebshihadeh) [reply selfdrive/ui/paint.cc:221]: how did you test that? I'm fine with just merging the change to started.

Comment (deanlee) [reply selfdrive/ui/paint.cc:221]: function update_lane_line_date has already call valid_frame_pt to make sure x,y>=0

Comment (adeebshihadeh) [reply selfdrive/ui/paint.cc:221]: makes sense

**Pull request merged**